redwood-compose/src/commonTest/kotlin/app/cash/redwood/compose/ChangeListenerTest.kt

```diff
@@ -59,11 +59,8 @@ class ProtocolChangeListenerTest : AbstractChangeListenerTest() {
     widgetSystem: TestSchemaWidgetSystem<WidgetValue>,
     snapshot: () -> T,
   ): TestRedwoodComposition<T> {
-    val state = DefaultProtocolState(
+    val compositionBridge = DefaultProtocolBridge(
       hostVersion = hostRedwoodVersion,
-    )
-    val compositionBridge = app.cash.redwood.protocol.guest.ProtocolBridge(
-      state = state,
       widgetSystemFactory = TestSchemaProtocolWidgetSystemFactory,
     )
     val widgetBridge = ProtocolBridge(
```

Comment:
Probably should be `hostBridge` and `guestBridge`. The existing `widgetBridge` likely just predates the pervasive use of host/guest terms.

Reply:
YES! Will do.